### PR TITLE
merge

### PR DESCRIPTION
**Pull request by ahlaamfaiz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,7 +56,15 @@
         </div>
       </div>
     </nav>
+<<<<<<< HEAD
+    <footer>
+      <div class="">
+        
+      </div>
+    </footer>
+=======
```
Comment (bushraaksoy) [inline index.html]: what is this <<<<HEAD

Comment (ahlaamfaiz) [reply index.html]: nishaidelete

Comment (bushraaksoy) [reply index.html]: okay

**Comment by ahlaamfaiz** [comment]
change in footer

File: styles/style.css [hunk 2]
```diff
@@ -27,3 +27,43 @@ body {
...
+.footer{
+  background-color:#4267b2;
+  color: white;
+  display: flex;
+  
+  /* gap: 50%; */
+}
+.footer h4 {
+  font-size: 20px;
+  margin: 0px 0px 8px ;
```
Comment (bushraaksoy) [inline styles/style.css]: margin should either have 2 or 4 numbers, it cant have three

**Review by bushraaksoy — COMMENTED** [review]
just change the margin and push it again.

File: styles/style.css [hunk 3]
```diff
@@ -65,5 +65,6 @@ body {
 }
 .copy{
   font-size: 10px;
-  margin: 0px 0px 16px;
+  margin: 0px 0px 16px 0px;
```
Comment (bushraaksoy) [inline styles/style.css]: you dont have to change it, but just letting you know that if ur adding for only one, you can just use margin-bottom: 16px, if you want

**Pull request merged**